PACKAGES.md

```diff
@@ -27,13 +27,13 @@ The dependencies between layers are enforced by the layer-check command._
 
 | Packages | Layer Dependencies |
 | --- | --- |
-| - [@fluidframework/driver-definitions](/packages/common/driver-definitions)</br>- [@fluidframework/odsp-driver-definitions](/packages/drivers/odsp-driver-definitions) | - [Core-Interfaces](#Core-Interfaces)</br>- [Protocol-Definitions](#Protocol-Definitions) |
+| - [@fluidframework/driver-definitions](/packages/common/driver-definitions)</br>- [@fluidframework/odsp-driver-definitions](/packages/drivers/odsp-driver-definitions) | - [Core-Interfaces](#Core-Interfaces)</br>&nbsp; |
 
 ### Container-Definitions
 
 | Packages | Layer Dependencies |
 | --- | --- |
-| - [@fluidframework/container-definitions](/packages/common/container-definitions)</br>&nbsp;</br>&nbsp; | - [Core-Interfaces](#Core-Interfaces)</br>- [Protocol-Definitions](#Protocol-Definitions)</br>- [Driver-Definitions](#Driver-Definitions) |
+| - [@fluidframework/container-definitions](/packages/common/container-definitions)</br>&nbsp; | - [Core-Interfaces](#Core-Interfaces)</br>- [Driver-Definitions](#Driver-Definitions) |
```

Comment:
@alexvy86 PACKAGES.md should be regenerated (`pnpm layer-check --md .`) after any addition/removal of a package, or any change to layerInfo.json.  Most engineers forget to do this, so this includes many unrelated updates from those omissions.  For random example, this line change originates from this PR here: https://github.com/microsoft/FluidFramework/pull/21200/files#diff-33f48800ac63780832d79c24ea32441d5b470c10b6cf9871116bac70a3925e7aR69